cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -1,4 +1,5 @@
 # Author: Jamie Strandboge <jamie@canonical.com>
+abi <abi/3.0>,
```

Comment:
This does not work on apparmor_parser 2.10.95 as seen on up-to-date ubuntu core 16